### ISSUE TITLE
[C#] remove accessibility concept

### ISSUE THREAD

**Issue by mikedamay** [exercism/v3#2720]
The accessibility concept should be removed from everywhere including:
- any design.md
- csharp/config.json
- concepts directory
- csharp/reference/exercise.json

If a concept such as encapsulation needs to be distinguished we can pick that up in a review of the track.

**Comment by mikedamay** [comment]
closed via PR #2733
